content/courses/advanced-ada/chapters/subprograms.rst

```diff
@@ -3,6 +3,146 @@ Subprograms
 
 .. include:: ../../global.txt
 
+.. _Operators:
+
+Operators
+---------
+
+Operators are commonly used for variables of scalar types such as
+:ada:`Integer` and :ada:`Float`. In these cases, they replace *usual* function
+calls. (To be more precise, operators are function calls, but written in a
+different format.) For example, we simply write :ada:`A := A + B + C;` when we
+want to add three integer variables. A hypothetical, non-intuitive version of
+this operation could be :ada:`A := Add (Add (A, B), C);`. In such cases,
+operators allow for expressing function calls in a more intuitive way.
+
+Many pre-defined operators exist for scalar types. Obviously, for record types,
```

Comment:
I think this intro should be more generalized: there are operators defined for other categories of types, such as arrays, eg the concatenation operator ("&"), and operators can be defined for any type in general, not just record and scalar types.

Reply:
Yes, I agree. I added some examples of operators for arrays. Let me know if I should add more examples.